File: ShowcaseView_java/sample/src/main/java/com/smorodov/showcaseview/sample/fragments/ShowcaseFragmentActivity.java
package com.smorodov.showcaseview.sample.fragments;

import android.os.Bundle;

import androidx.fragment.app.FragmentActivity;

import com.smorodov.showcaseview.sample.R;


public class ShowcaseFragmentActivity extends FragmentActivity {

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.fragment_activity_layout);
    }

}

File: ShowcaseView_kotlin/sample/src/main/java/com/smorodov/showcaseview/sample/fragments/ShowcaseFragmentActivity.kt
package com.smorodov.showcaseview.sample.fragments

import android.os.Bundle
import androidx.fragment.app.FragmentActivity
import com.smorodov.showcaseview.sample.R

class ShowcaseFragmentActivity : FragmentActivity() {
    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.fragment_activity_layout)
    }
}